src/dirac_cwl_proto/job/json_catalog.py

```diff
@@ -0,0 +1,37 @@
+
+from typing import Annotated
+from pydantic import BaseModel, RootModel, BeforeValidator, AnyUrl
+
+
+def validateLFN(value: str) -> str:
+    value = value.removeprefix("LFN:")
+    return value
+
+def validatePFN(value: str) -> str:
+    value = value.removeprefix("PFN:")
+    return value
+
+# Logical File Name such as LFN:/lhcb/MC/2024/HLT2.DST/00327923/0000/00327923_00000533_1.hlt2.dst
+LFN = Annotated[str, BeforeValidator(validateLFN)]
+PFN = Annotated[AnyUrl, BeforeValidator(validatePFN)]
+StorageElementId = str
+ChecksumType = str
+FileChecksum = str
+
+
+class JSONCatalog(RootModel):
```

Comment:
I'm just wondering if it's the best name we can come up.
What about `ReplicaCatalog`?